examples/model_configs/base_model.yaml

```diff
@@ -0,0 +1,12 @@
+model:
+  type: "base" # can be base, tgi, or endpoint
+  base_params:
+    model_args: "pretrained=HuggingFaceH4/zephyr-7b-beta,revision=main" # pretrained=model_name,trust_remote_code=boolean,revision=revision_to_use,model_parallel=True ...
```

Comment:
Would be nice to have the name, revision etc.. as fields in the yaml wdyt ?

Reply:
I thought about it, but 1) keeping model_args allows us to be homogeneous with the cli for now 2) using the dict gives us some flexibility if we want to edit model params in the future